frontend/lib/norent/letter-builder/tests/national-metadata.test.tsx

```diff
@@ -54,9 +54,14 @@ describe("getNorentMetadataForUSState()", () => {
       expect(typeof md.lawForBuilder.linkToLegislation).toMatch(
         /string|undefined/
       );
-      expect(typeof md.lawForBuilder.textOfLegislation).toBe("string");
+      expect(typeof md.lawForBuilder.textOfLegislation).toMatch(
+        /string|undefined/
+      );
+      if (!md.lawForBuilder.stateWithoutProtections) {
+        expect(typeof md.lawForBuilder.textOfLegislation).toBe("string");
+        expect(md.lawForLetter.textOfLegislation.length).toBeGreaterThan(0);
+      }
```

Comment:
This test follows the same logic as your implementation of the Know Your Rights page— if it's a state without protections, we don't render any legislation.

Reply:
ah noooooooice thanks for updating the tests to reflect this!